index.js

```diff
@@ -148,7 +148,7 @@ function runTenonIO(context) {
 function runChromeDevTools(context) {
 
   var data = fs.readFileSync(AUDIT_FILE, 'utf-8');
-  data = data + ' return axs.Audit.run();';
+  data = data + ' var configuration = new axs.AuditConfiguration(' + JSON.stringify(context.config.chromeA11YDevTools.auditConfiguration) + '); return axs.Audit.run(configuration);';
```

Comment:
@sjelin Tests that haven't set this option will run exactly the same. Passing in a configuration object with an empty constructor axs.Audit.run produces the same output as just calling axs.Audit.run(); 

Here's a jsfiddle as an example:
https://jsfiddle.net/jamsea/ejv52v5r/1/